src-java/reroute-topology/reroute-messaging/src/main/java/org/openkilda/messaging/info/reroute/error/GlobalTimeoutError.java

```diff
@@ -0,0 +1,30 @@
+/* Copyright 2020 Telstra Open Source
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+
+package org.openkilda.messaging.info.reroute.error;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+
+@Data
+@EqualsAndHashCode(callSuper = true)
+public class GlobalTimeoutError extends RerouteError {
+    @JsonCreator
+    public GlobalTimeoutError() {
+        super("Global timeout of retry for flow");
```

Comment:
As for me word "retry" is a little bit confusing here. Maybe it was first reroute. My suggest is to change message to something like "Flow reroute operation is timed out" or "Flow reroute operation reached global timeout".

Reply:
it's typo, not retry but reroute, fixed